pages/audits.js

```diff
@@ -11,13 +12,34 @@ import FetchRequest from "../lib/fetch-request";
 import { snakeCaseToHuman } from "../lib/general";
 
 export default function AuditDoc({ auditDoc }) {
-  const keyedAudits = Object.keys(auditDoc).map((key) => {
-    return {
-      key: key.split(".")[2],
-      ...auditDoc[key],
-    };
+  const result = _.flatMap(auditDoc, (auditGroup, key) => {
+    return auditGroup.map((audit) => {
+      const newKeys = key.split(".")[2];
+      return { ...audit, newKeys };
+    });
   });
-  const auditsGroupedByCollection = _.groupBy(keyedAudits, "key");
+  const auditsGroupedByCollection = _.groupBy(result, "newKeys");
+
+  const auditKeyColor = [
+    {
+      audit_level: "ERROR",
+      audit_description: "Incorrect or inconsistent metadata",
+    },
+    {
+      audit_level: "NOT_COMPLIANT",
+      audit_description:
+        "Not fulfilling a compliance standard like attached documents",
+    },
+    {
+      audit_level: "WARNING",
+      audit_description:
+        "Possibly inconsistent metadata. Data can be released with warnings",
+    },
+    {
+      audit_level: "INTERNAL_ACTION",
+      audit_description: "Metadata errors that require DACC staff to resolve",
+    },
+  ];
```

Comment:
Since `auditKeyColor` never changes, it’s best to put these outside of functions so that it only gets executed on page load. Putting it in the function means it gets executed every render.

Reply:
ok